sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/tests/ManagementApiTests/ApiRevisionTests.cs

```diff
@@ -282,12 +282,12 @@ public async Task CreateListUpdateDelete()
                         "*",
                         deleteRevisions: true);
 
-                    testBase.client.ApiRelease.Delete(
-                        testBase.rgName,
-                        testBase.serviceName,
-                        newApiId,
-                        newReleaseId,
-                        "*");
+                    //testBase.client.ApiRelease.Delete(
+                    //    testBase.rgName,
```

Comment:
why is it commented

Reply:
it's been not found for a while https://msazure.visualstudio.com/DefaultCollection/One/_build/results?buildId=70266469&view=ms.vss-test-web.build-test-results-tab